spec/migrations/drop_think_feel_do_dashboard_moderators_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
```

Comment:
While this spec is clever, I would probably drop it. There's no reason to have it run every time we run specs, as the migration is only intended to be run a couple of times at most.